### DISCUSSION TITLE
How to enable the s3 connectivity ?

### DISCUSSION THREAD

**Discussion by Vishwavijet** [structurizr/onpremises#91] [Q&A] [answered]
Im going through the docs mentioned here
https://docs.structurizr.com/onpremises/data-storage

But didnt get exactly where the properties needs to be changed.
Im using the docker image and passing these keys as configmap to the deployment and yet I see 
`ContextLoaderListener - Data directory: /usr/local/structurizr`

Please do help on this.

**Comment by RJanssens** [comment, marked as answer]
You should create a structurizr.properties-file with storage properties (& other details if needed, like authentication provider) and put it in your data directory.
